### DISCUSSION TITLE
Context question

### DISCUSSION THREAD

**Discussion by andrconstruction** [react-hook-form/react-hook-form#1928] [Q&A] [answered]
No returned data from nested forms instead returns syntetic event. should I use methods inside nested forms like methods.error or something like this? Have you more detailed or worked in the sandbox an example of context conception, when  parts of form place in other files

**Comment by bluebill1049** [comment, marked as answer]
i need more context with your question as well.

**Reply by andrconstruction** [reply]
Thank you? But figured it out myself )

**Reply by bluebill1049** [reply]
awsome.
